ApplicationLibCode/ProjectDataModel/WellLog/RimWellLogFile.cpp

```diff
@@ -78,3 +88,34 @@ std::vector<RimWellLogFileChannel*> RimWellLogFile::wellLogChannels() const
     }
     return channels;
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QDateTime RimWellLogFile::date() const
+{
+    return m_date;
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimWellLogFile::fieldChangedByUi( const caf::PdmFieldHandle* changedField, const QVariant& oldValue, const QVariant& newValue )
+{
+    if ( changedField == &m_date )
+    {
+        // Due to a possible bug in QDateEdit/PdmUiDateEditor, convert m_date to a QDateTime having UTC timespec
+        m_date = RiaQDateTimeTools::createUtcDateTime( m_date().date(), m_date().time() );
+    }
+}
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimWellLogFile::defineEditorAttribute( const caf::PdmFieldHandle* field, QString uiConfigName, caf::PdmUiEditorAttribute* attribute )
+{
+    caf::PdmUiDateEditorAttribute* attrib = dynamic_cast<caf::PdmUiDateEditorAttribute*>( attribute );
```

Comment:
Can be combined into if(auto* attrib = dynamic_ca....

Reply:
Fixed.